src/main/java/se/sundsvall/supportmanagement/service/mapper/ErrandMapper.java

```diff
@@ -48,7 +48,8 @@ public static ErrandEntity toErrandEntity(String namespace, String municipalityI
 			.withStakeholders(toStakeholderEntities(errandEntity, errand.getStakeholders()))
 			.withStatus(errand.getStatus())
 			.withTitle(errand.getTitle())
-			.withType(errand.getClassification().getType());
+			.withType(errand.getClassification().getType())
```

Comment:
Not introduced in this PR, but still:

Remove declaration of `errandEntity`-variable and just return the errand directly? 

E.g.
```
return ErrandEntity.create()
  .withAssignedGroupId(errand.getAssignedGroupId())
  .with...
```

Reply:
Not my creation but`.withStakeholders(toStakeholderEntities(errandEntity, errand.getStakeholders()))` relies on the entity being declared before. So just removing the declaration and returning the object directly isn't an option in this case.

Reply:
Ahh... I see this now.